docs/products/kafka/howto/viewing-resetting-offset.rst

```diff
@@ -0,0 +1,145 @@
+View and reset consumer group offsets
+============================================
+
+The `open source Apache Kafka code <https://kafka.apache.org/downloads>`_ includes a ``kafka-consumer-groups.sh`` utility enabling you to view and manipulate the state of consumer groups. 
+Before using the tool, few setup steps need to be performed.
+
+Define the configuration file
+-----------------------------
+
+#. Create the Java keystore and truststore for your Aiven for Apache Kafka service using the :ref:`dedicated Aiven CLI command <avn_service_user_kafka_java_creds>`.
+
+#. Create a ``consumer.properties`` properties file pointing to the keystore and truststore files with the following entries:
```

Comment:
"keystore and truststore files" - to avoid saying word "file" again, can we replace it with "keystore and truststore certificates", those files are certificates, right?

Reply:
well... nope, they are files enclosing the certificates, are like lockers with the keys inside